Makefile.am

```diff
@@ -500,14 +506,13 @@ NETDATA_COMMON_LIBS = \
 	$(OPTIONAL_JUDY_LIBS) \
 	$(OPTIONAL_SSL_LIBS) \
 	$(NULL)
-# TODO: Find more graceful way to add libs for AWS Kinesis
 
 sbin_PROGRAMS += netdata
 netdata_SOURCES = $(NETDATA_FILES)
 netdata_LDADD = \
 	$(NETDATA_COMMON_LIBS) \
 	$(NULL)
-if ENABLE_BACKEND_KINESIS
+if ENABLE_CXX_LINKER
```

Comment:
There is a ENABLE_BACKEND_KINESIS on Makefile.am.
should it be changed too ?

Reply:
No. We need ENABLE_CXX_LINKER for both Kinesis and Remote Write backends. ENABLE_BACKEND_KINESIS is used specifically for Kinesis dependencies.

Reply:
ah yes of course, i rushed in, thanks!